packages/react/src/useEditor.ts

```diff
@@ -113,14 +113,121 @@ export const useEditor = (options: Partial<EditorOptions> = {}, deps: Dependency
 
     return () => {
       isMounted = false
+      editorRef.current?.destroy()
```

Comment:
This also should fix the memory leak concerns from my understanding following the change to ref here